connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorsResource.java

```diff
@@ -89,6 +89,9 @@ public ConnectorsResource(Herder herder) {
     public Response createConnector(final @QueryParam("forward") Boolean forward,
                                     final CreateConnectorRequest createRequest) throws Throwable {
         String name = createRequest.name();
+        if (name.contains("/")) {
+            throw new BadRequestException("connector name should not contain '/'");
```

Comment:
well the PUT API takes a PathParam so it won't allow for a slash in the connector name. I agree it is not explicit but no connector with a slash in its name can be PUT